common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -991,76 +991,75 @@ def test_save_problem_submitted_no_randomize(self, rerandomize):
 
     def test_check_button_name(self):
```

Comment:
I'd say this test case can be simplified quite a bit since the button never change its name anymore to "Final ...".